src/grpc/clients/ea_restaurant_java_etl_grpc_client.py

```diff
@@ -47,7 +49,43 @@ def insert_mongo_order_status_histories_from_python_etl(
                 mongoOrderStatusHistory=mongo_order_status_histories
             )
         )
-        metadata = (("authorization", access_token),)
+        metadata = (
+            ("authorization", generate_bearer_token_from_access_token(access_token)),
+        )
         return self.mongo_order_status_service_stub.insertMongoOrderStatusHistoriesFromPythonEtl(
             mongo_order_status_histories_request, metadata=metadata
         )
+
+    def execute_service_with_oauth2_auth(self, grpc_call, login_client, refresh_token):
```

Comment:
you don't need to send `login_client` neither `refresh_token`, this should live inside the client instead.  No one else except for this client will hit the java etl server

Reply:
Done.